src/com/lftechnology/java/training/sanish/stemleaf/UserInput.java

```diff
@@ -0,0 +1,44 @@
+package com.lftechnology.java.training.sanish.stemleaf;
+
+import java.util.Scanner;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Provide methods to get inputs form console
+ * 
+ * @author Sanish Maharjan <sanishmaharjan@lftechnology.com>
+ */
+public class UserInput {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)